operator/src/main/resources/Operator.properties

```diff
@@ -177,6 +177,8 @@ WLSDO-0024=More than one item under ''spec.commonMountVolumes'' in the domain re
 WLSDO-0025=More than one item under ''spec.commonMountVolumes'' in the domain resource has same DNS-1123 name ''{0}''.
 WLSDO-0026=An item under ''spec.commonMountVolumes'' in the domain resource does not have a name. A name is required for \
   the common mount volumes defined under ''spec.commonMountVolumes''.
+WLSDO-0027=Monitoring Exporter port ''{0}'' conflicts with WebLogic Server instance ''{1}'' port ''{2}''
+WLSDO-0028=Monitoring Exporter port ''{0}'' conflicts with Cluster ''{1}'' dynamic server template port ''{2}''
```

Comment:
IMO, it'd be helpful if these two messages mentioned the corresponding domain resource setting.

Reply:
Sure, I can do that.